server/clients/eth/helper.go

```diff
@@ -0,0 +1,24 @@
+package goethclient
+
+import (
+	"math"
+	"math/big"
+)
+
```

Comment:
Do these functions belong to eth? The first one I would think yes but then the second one we might need it in tfchain too no? Maybe this code belongs in server/pkg?

Reply:
can move later if needed, right now it's not used